### DISCUSSION TITLE
I made a colorpicker for ace editor.

### DISCUSSION THREAD

**Discussion by easylogic** [ajaxorg/ace#4725] [Ideas]
Hi  everyone.

I made a colorpicker for ace editor. 
I'd love to hear your opinions.

It's open source and the code is here.

https://github.com/easylogic/ace-colorpicker 

A simple sample can be found here. 

https://codepen.io/easylogic/pen/RwVOGed

Thanks. 

ps. 

screen shot 

<img width="877" alt="스크린샷 2021-08-18 오후 9 19 36" src="https://user-images.githubusercontent.com/591983/129898401-7f78c178-4ec0-4d95-94d4-39450d552794.png">

**Comment by brynne8** [comment]
@nightwing I think this extension is worth adding to Wiki or website.

**Comment by WinterSilence** [comment]
wow! great solution!
I'm can't find my old implementation, but extension worked around `<input type="color">`, RBG/RGBA converting to hexadecimal value.
*update* https://regex101.com/library/dVOwn0

**Comment by ldijkman** [comment]
Hello

Nice Verry handy, forked your codepen

but having trouble to get it in my esp8266 esp32 webserver ace js editor code ;-(
<a href ="https://github.com/ajaxorg/ace/discussions/4836">github.com/ajaxorg/ace/discussions/4836</a>
https://github.com/ajaxorg/ace/discussions/4836

---

ok got it a bit working
works only in your head style
not in your body

full page view https://codepen.io/ldijkman/full/LYdZpYp
https://codepen.io/ldijkman/pen/LYdZpYp

![2022-07-10-145229_1920x1080_scrot](https://user-images.githubusercontent.com/45427770/178147017-845cd84a-0029-4cdc-8ae8-54b3ce39ad0f.png)
